### PR TITLE
COM-184: Display Complete Path label for the home site

### PR DESCRIPTION
**Pull request by jennyvivid**
<details>
 <summary>Screenshot</summary>
 
Screenshot 2023-11-20 at 13 22 59

</details>

I'm not sure how 'ugly' the code repetition for the FieldContainer is here.

### CODE REVIEW

File: packages/admin/cms-admin/src/pages/createEditPageNode.tsx [hunk 1]
```diff
@@ -313,7 +314,19 @@ export function createEditPageNode({
...
+                                                    <FieldContainer
+                                                        label={intl.formatMessage({
+                                                            id: "comet.pages.pages.page.path",
+                                                            defaultMessage: "Complete Path",
+                                                        })}
+                                                        variant="horizontal"
+                                                    >
+                                                        <Typography>/</Typography>
+                                                    </FieldContainer>
+                                                </>
```
Comment (thomasdax98) [inline packages/admin/cms-admin/src/pages/createEditPageNode.tsx:328]: Do you need that `if`? Can't you do that down below in the return?

Somewhat like 

```tsx
<FieldContainer
    label={intl.formatMessage({
        id: "comet.pages.pages.page.path",
        defaultMessage: "Complete Path",
    })}
    variant="horizontal"
>
    <Typography>
        {values.slug === "home" ? "/" : `${parentPath}/${values.slug}`}
    </Typography>
</FieldContainer>
```

Comment (thomasdax98) [reply packages/admin/cms-admin/src/pages/createEditPageNode.tsx:328]: Also the fragment is not necessary because you're only returning one element

**Comment by thomasdax98** [comment]
Also please change PRs from "Draft" to "Ready for review" once you want us to review them

**Review by johnnyomair — CHANGES_REQUESTED** [review]
Please change as suggested by @thomasdax98

File: packages/admin/cms-admin/src/pages/createEditPageNode.tsx (hunk 1 above)
Comment (jennyvivid) [reply packages/admin/cms-admin/src/pages/createEditPageNode.tsx:328]: I wasn't sure whether the code after the FieldContainer should also run when it's _home_, so I left the _if_ for the time being just to be safe. Will do 👍

**Review by johnnyomair — APPROVED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Comment by thomasdax98** [comment]
@johnnyomair I changed the base to main. Please reapprove

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**